kiwi/xml_state.py

```diff
@@ -2110,6 +2110,44 @@ def get_root_filesystem_uuid(self) -> Optional[str]:
         """
         return self.root_filesystem_uuid
 
+    def get_archives_target_dirs(
+        self, packages_sections_names: Optional[List[xml_parse.packages]]
+    ) -> Dict:
```

Comment:
This can be static

```python
@staticmethod
def get_archives_target_dirs(
    packages_sections_names: Optional[List[xml_parse.packages]]
) -> Dict:
```

Reply:
to fix the codacy issue report